src/app/(proper_react)/redesign/(authenticated)/user/dashboard/View.tsx

```diff
@@ -197,6 +198,22 @@ export const View = (props: Props) => {
     }
   }
 
+  // Fixed in: MNTOR-2011
+  const freeScanCta = isScanResultItemsEmpty ? (
+    <p>
+      {l10n.getFragment("dashboard-exposures-all-fixed-free-scan", {
+        vars: {
+          data_broker_total_num: ONEREP_DATA_BROKER_COUNT,
+        },
+        elems: {
+          free_scan_link: <a href="/redesign/user/welcome" />,
```

Comment:
I assume you need to update this as well?

Reply:
@flodolo which line are you referring to? 209?